src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/BOM/MaterialForm/MaterialFormView.tsx

```diff
@@ -157,7 +158,12 @@ const MaterialFormView: React.FC<MaterialFormViewProps> = ({
         </Grid>
         <Grid item xs={6}>
           <FormControl fullWidth>
-            <FormLabel>Manufacturer Part Number</FormLabel>
+            <FormLabel>
+              Manufacturer Part Number
+              <Tooltip title={'Enter \'N/A\' if no Manufacturer Part Number'} placement="right">
+                <HelpIcon style={{ marginBottom: '-0.2em', fontSize: 'medium', marginLeft: '5px', color: 'lightgray' }} />
```

Comment:
The tooltip should be on the left side of the Manufacturer Part Number label, right now it is on the right